stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/DropdownFieldUI.kt

```diff
@@ -162,23 +158,13 @@ internal fun DropDown(
 
         DropdownMenu(
             expanded = expanded,
-
-            // We will show up to two items before
-            initialFirstVisibleItemIndex = if (selectedIndex >= 1) {
-                min(
-                    max(selectedIndex - 2, 0),
-                    max(selectedIndex - 1, 0)
-                )
-            } else {
-                selectedIndex
-            },
```

Comment:
Removed this in favor of the default Compose `DropdownMenu`. It'll will automatically show items before without needing this.